phones/apps.py

```diff
@@ -19,6 +19,11 @@ def __init__(self, app_name, app_module):
             self._twilio_client = Client(
                 settings.TWILIO_ACCOUNT_SID, settings.TWILIO_AUTH_TOKEN
             )
+            # Create TwiML app to access its sms_status_callback
+            # Note:
+            self.twiml_app = self._twilio_client.applications(
+                settings.TWILIO_SMS_APPLICATION_SID
+            ).fetch()
```

Comment:
This works for me. My `TWILIO_SMS_APPLICATION_SID` is set to my TwiML app ID (which is under Voice in the Twilio menu), and `twiml.sms_status_callback` is `/api/v1/sms_status` on my ngrok.io domain. So that's not the issue with my integration testing...